conf/defaults.ini

```diff
@@ -1013,6 +1013,12 @@ reset_basic_roles = false
 # Validate permissions' action and scope on role creation and update
 permission_validation_enabled = true
 
+# List of resources to generate managed permissions for upon creation (dashboard, folder, service-account, datasource)
+managed_permissions_on_creation = dashboard, folder, service-account, datasource
+
+# List of resources to seed managed permission wildcards for (dashboard, folder, datasource)
+managed_permissions_wildcard_seeds =
```

Comment:
We could hide this from the config altogether and only communicate about it to users for whom we think this is relevant.